cmd/pilot-agent/main.go

```diff
@@ -111,9 +115,13 @@ func init() {
 		"Sidecar proxy unique ID. If not provided uses ${POD_NAME}.${POD_NAMESPACE} environment variables")
 	proxyCmd.PersistentFlags().StringVar(&sidecar.Domain, "domain", "cluster.local",
 		"DNS domain suffix")
+	proxyCmd.PersistentFlags().StringVar(&customConfig, "envoyconfig", "",
+		"Custom envoy config")
 
 	cmd.AddFlags(rootCmd)
 
+	rootCmd.Flags().AddGoFlagSet(flag.CommandLine)
```

Comment:
_Force_ is probably too strong a word. `cmd.AddFlags` changes the default behavior to write to stderr instead of defaulting to a tmp file. The user should still be able to write to file with `--logtostderr=false`.